packages/preferences/src/components/preference-toggle-menu-item/index.js

```diff
@@ -23,6 +23,7 @@ export default function PreferenceToggleMenuItem( {
 	handleToggling = true,
 	onToggle = () => null,
 	disabled = false,
+	disableSpeech = false,
```

Comment:
We should be conservative when adding props to components and especially public ones, like this one. My comment related to `handleToggling` was about the same thing - it shouldn't be added in the first place.

>it makes most sense to just use MenuItem component directly and avoid the weird handleToggling and messageActivated/messageDeactivated messages.

`PreferenceToggleMenuItem` in essence is a thin wrapper of `MenuItem` for toggling preferences and speak the messages. In these cases we handle both of these in the action, so there is no need to use `PreferenceToggleMenuItem` at all, and instead use `MenuItem` directly.